testsuite/features/support/twopence_init.rb

```diff
@@ -38,12 +38,14 @@
   next if node.nil?
 
   hostname, _local, _remote, code = node.test_and_store_results_together('hostname', 'root', 500)
-  raise 'Cannot get hostname for node' if code.nonzero?
+  raise 'Cannot get hostname for node' if code.nonzero? || hostname.empty?
```

Comment:
Should this tell  use the node name?

Reply:
@juliogonzales ideally, yes, but we don't know it.